src/browser/components/SettingsPage.jsx

```diff
@@ -434,154 +263,11 @@ export default class SettingsPage extends React.Component {
     });
   }
 
-  setDarkMode() {
-    this.setState({
-      isDarkMode: this.props.setDarkMode(),
-    });
-  }
-
-  handleClose = () => {
-    const win = remote.getCurrentWindow();
-    win.close();
-  }
-
-  handleMinimize = () => {
-    const win = remote.getCurrentWindow();
-    win.minimize();
-  }
-
-  handleMaximize = () => {
-    const win = remote.getCurrentWindow();
-    win.maximize();
-  }
-
-  handleRestore = () => {
-    const win = remote.getCurrentWindow();
-    win.restore();
-  }
-
-  openMenu = () => {
-    // @eslint-ignore
-    this.threeDotMenu.current.blur();
-    this.props.openMenu();
-  }
-
-  handleDoubleClick = () => {
-    const doubleClickAction = remote.systemPreferences.getUserDefault('AppleActionOnDoubleClick', 'string');
-    const win = remote.getCurrentWindow();
-    if (doubleClickAction === 'Minimize') {
-      win.minimize();
-    } else if (doubleClickAction === 'Maximize' && !win.isMaximized()) {
-      win.maximize();
-    } else if (doubleClickAction === 'Maximize' && win.isMaximized()) {
-      win.unmaximize();
-    }
-  }
-
-  handleMaximizeState = () => {
-    const win = remote.getCurrentWindow();
-    this.setState({maximized: win.isMaximized()});
-  }
-
-  handleFullScreenState = () => {
-    const win = remote.getCurrentWindow();
-    this.setState({fullScreen: win.isFullScreen()});
+  backToIndex = (index) => {
+    remote.getCurrentWindow().send('toggle-settings-page', index);
   }
 
   render() {
-    const tabsRow = (
-      <TabBar
-        id='tabBar'
-        isDarkMode={this.state.isDarkMode}
-        teams={[]}
-        showAddServerButton={false}
-      />
-    );
-
-    let topBarClassName = 'topBar';
-    if (process.platform === 'darwin') {
-      topBarClassName += ' macOS';
-    }
-    if (this.state.isDarkMode) {
-      topBarClassName += ' darkMode';
-    }
-    if (this.state.fullScreen) {
-      topBarClassName += ' fullScreen';
-    }
-
-    let maxButton;
-    if (this.state.maximized) {
-      maxButton = (
-        <div
-          className='button restore-button'
-          onClick={this.handleRestore}
-        >
-          <img src={restoreButton}/>
-        </div>
-      );
-    } else {
-      maxButton = (
-        <div
-          className='button max-button'
-          onClick={this.handleMaximize}
-        >
-          <img src={maximizeButton}/>
-        </div>
-      );
```

Comment:
Handled in `MainPage`. Not to worry, this UI feature has been maintained in this fork. See `settings.css` and logic added to `TabBar` component in `MainPage`.